### DISCUSSION TITLE
How to set the number of Jobs when running with FillingModeFlag = True

### DISCUSSION THREAD

**Discussion by marianne013** [DIRACGrid/DIRAC#6117] [Q&A] [answered]
It used to be MaxJobsInFillMode in the SiteDirector (and confusingly that option still seems to be there on the certification server), but that option seems to have been removed in v7r1. How is this meant to work these days ?

**Comment by fstagni** [comment, marked as answer]
The option was removed in DIRAC v7r1p26 through https://github.com/DIRACGrid/DIRAC/pull/4926 as not fitting properly with PoolCE logic. The maximum number of jobs in filling mode can be indirectly set by specifying the number of cycles of the JobAgent. 
This can be set, CEType by CEType, or CE by CE or queue by queue with `ExtraPilotOptions = -M x` where `x` is the number of cycles (which is 10 by default).

Note that Pilots hardcode a bunch of options if  `CEType==Pool`: https://github.com/DIRACGrid/Pilot/blob/master/Pilot/pilotCommands.py#L985-L992

**Reply by marianne013** [reply]
Thanks
